mergify/README.md

```diff
@@ -0,0 +1,61 @@
+# Agent Check: Mergify
+
+## Overview
+
+This check tracks [Mergify][1] merge queue statistics.
+
+## Setup
+
+### Installation
+
+To install the Mergify check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
+ on any machine.
+
+2. Run `ddev release build mergify` to build the package.
+
+3. [Download the Datadog Agent][2].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/mergify/dist/<ARTIFACT_NAME>.whl`.
+
+### Configuration
+
+1. Edit the `mergify.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][3] to start collecting your Mergify [metrics](#metrics). See the [
```

Comment:
Looks like this step may still be under development -- could you please confirm and update?

Reply:
I re-added the missing sentence.